e2e-polybft/framework/test-cluster.go

```diff
@@ -198,13 +212,14 @@ func NewTestCluster(t *testing.T, validatorsCount int, opts ...ClusterOption) *T
 	require.NoError(t, err)
 
 	config := &TestClusterConfig{
-		t:           t,
-		WithLogs:    isTrueEnv(envLogsEnabled),
-		WithStdout:  isTrueEnv(envStdoutEnabled),
-		TmpDir:      tmpDir,
-		Binary:      resolveBinary(),
-		EpochSize:   10,
-		EpochReward: 1,
+		t:             t,
+		WithLogs:      isTrueEnv(envLogsEnabled),
+		WithStdout:    isTrueEnv(envStdoutEnabled),
+		TmpDir:        tmpDir,
+		Binary:        resolveBinary(),
+		EpochSize:     10,
+		EpochReward:   1,
+		BlockGasLimit: 1e7, // 10M
```

Comment:
It turned out that default gas limit of 0x500000 (~5.2M) wasn't enough to include register validator transaction, therefore I've increased it to 10M:
```go
2022-12-30T12:00:31.168+0100 [ERROR] polygon.txpool: failed to add tx: err="exceeds block gas limit"
2022-12-30T12:00:31.168+0100 [ERROR] polygon.server.dispatcher: failed to dispatch: method=eth_sendRawTransaction err="exceeds block gas limit"
```